hazelcast/src/main/java/com/hazelcast/spi/discovery/integration/DiscoveryService.java

```diff
@@ -53,13 +57,13 @@
      *
      * @return a set of discovered and filtered nodes
      */
-    Iterable<DiscoveredNode> discoverNodes();
+    Iterable<DiscoveryNode> discoverNodes();
 
     /**
      * The <tt>start</tt> method is called on system startup to implement simple
      * lifecycle management. This method is expected to call
-     * {@link DiscoveryStrategy#destroy()} on all discovered and activated strategies
-     * before the service itself will shutdown.
+     * {@link DiscoveryStrategy#destroy()} on all discovered
+     * and activated strategies before the service itself will shutdown.
```

Comment:
I would align the name of the method with the documentation. So either talk about destroy in the documentation, or rename the method to shutdown.